sourcegraph/sourcegraph
# sg: Make sg setup prompt choices clearer

The usage of "you" and "I" in the prompts in `sg setup` can be a bit unclear for first time users.

This change makes the `sg setup` choices between automatic and manual fix clearer by making it more explicit what the choices will do.

## Test plan

- Run `sg setup`
- Setup choice prompts should be updated
